build.gradle

```diff
@@ -647,6 +647,9 @@ subprojects {
       "-Xlint:unused"
     ]
 
+    if (versions.baseScala == '2.13')
+      scalaCompileOptions.additionalParameters += ["-Wconf:msg=@nowarn annotation does not suppress any warnings:s"] // See https://github.com/scala/scala/pull/9960
```

Comment:
Why do we have to do this instead of removing the suppressions that are no longer needed?